src/requestBuilder.js

```diff
@@ -29,14 +29,26 @@ var requestBuilder = {
       params: requestBuilder._getHitsSearchParams(state)
     });
 
-    // One for each disjunctive facets
+    // One for each disjunctive facet
     state.getRefinedDisjunctiveFacets().forEach(function(refinedFacet) {
       queries.push({
         indexName: index,
         params: requestBuilder._getDisjunctiveFacetSearchParams(state, refinedFacet)
       });
     });
 
+    // One for each conjunctive facet
```

Comment:
@dhayab I'm I correct understanding 

1. that we're sending additional search request. In the case N additional request(s) where N is the number of facet(s) in the InstantSearch instance, and which have at least one active / refined facet value? 
2. that for those extra request we retrieve 1000 values, increasing at the same time the response payload?

Reply:
That's exactly that @seafoox.

Those are additional queries agregated in a single network request, thus having no pricing impact on the customer. The search parameters for those queries set `hitsPerPage: 0` because we don't need that, but they can indeed include up to 1000 facet values each.